### PR TITLE
Changed cell radius when mouse hovers over

### LINKED ISSUE #1633: Make radius to detect cells under the cursor bigger
To make it easier to hover over small cells, and partly mitigate: https://github.com/Revolutionary-Games/Thrive/issues/1394

### PR DESCRIPTION
**Pull request by Straywind**
closes #1633

### CODE REVIEW

File: simulation_parameters/Constants.cs [hunk 1]
```diff
@@ -412,6 +412,11 @@ public static class Constants
     public const float TOOLTIP_DEFAULT_DELAY = 1.0f;
 
     public const float TOOLTIP_FADE_SPEED = 0.1f;
+    
+    /// <summary>
+    ///   Increment to ai membrane microbes to make hovering over small cells easier
```
Comment (hhyyrylainen) [inline simulation_parameters/Constants.cs:417]: I feel like this should be reworded a bit as this isn't really an increment to the AI membrane, it's extra radius added when checking mouse hovered microbes. Also AI should be upper case.

File: simulation_parameters/Constants.cs [hunk 2]
```diff
@@ -412,6 +412,11 @@ public static class Constants
...
+    ///</summary>
+    public const float MEMBRANE_INCREMENT_RADIUS_FOR_MOUSE_HOVER = 2.0f;
```
Comment (hhyyrylainen) [inline simulation_parameters/Constants.cs:419]: I think that a name like `MICROBE_HOVER_DETECTION_EXTRA_RADIUS` would be more descriptive here.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
Left a few review comments. Also if you missed it, CI is not happy about some trailing whitespace, which you'll need to fix in order for the PR to no longer have the red x next to it.

File: simulation_parameters/Constants.cs [hunk 3]
```diff
@@ -412,6 +412,11 @@ public static class Constants
     public const float TOOLTIP_DEFAULT_DELAY = 1.0f;
 
     public const float TOOLTIP_FADE_SPEED = 0.1f;
+    
+    /// <summary>
+    ///   Increment to AI membrane microbes to make hovering over small cells easier
```
Comment (hhyyrylainen) [inline simulation_parameters/Constants.cs:417]: I still find this comment confusing / it doesn't correctly explain what this variable is for.
Also CI is still unhappy about the trailing whitespace.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

**Comment by Straywind** [comment]
Yes, yes, I'm working on it. Got called for lunch halfway through.

**Comment by Straywind** [comment]
Better, or too wordy?

**Review by hhyyrylainen — APPROVED** [review]

**Comment by hhyyrylainen** [comment]
I didn't test this but the change looks correct in terms of what it is supposed to do. And I trust you did at least basic testing on your changes.
